pages/management/PKPToolsHandler.inc.php

```diff
@@ -22,7 +22,7 @@
 use APP\i18n\AppLocale;
 use APP\notification\NotificationManager;
 use APP\template\TemplateManager;
-use PKP\components\PKPStatsJobsTable;
+use Illuminate\Support\Facades\Queue;
```

Comment:
This is being used?

Reply:
Sorry, test code that I forgot to remove before committing.  Can you remove  it after merging?